src/modules/paymentProcessor/bridging/ConnextBridgeLogic.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.23;
+
+import {CrosschainBase_v1} from
+    "src/modules/paymentProcessor/bridging/abstracts/CrosschainBase_v1.sol";
+import {IERC20PaymentClientBase_v1} from
+    "@lm/interfaces/IERC20PaymentClientBase_v1.sol";
+import {IERC20} from "@oz/token/ERC20/IERC20.sol";
+
+interface IEverclearSpoke {
+    function newIntent(
+        uint32[] memory destinations,
+        address to,
+        address inputAsset,
+        address outputAsset,
+        uint amount,
+        uint24 maxFee,
+        uint48 ttl,
+        bytes memory data
+    ) external returns (bytes32 intentId, uint amountOut);
+}
+
+contract ConnextBridgeLogic is CrosschainBase_v1 {
```

Comment:
Why did you went the route of inheriting from the crosschain base? This would make this contract a module (as the base inherits from `Module_v1`?

It seems to me that this logic is very Connext/Everclear specific so it could be added to the Connext Implementation

Reply:
Just went back to the diagram to understand if this is something you outlined in it.

My assumption was that the `ConnextBridgeLogic` is something provided by Connext and not a contract we create ourselves

Reply:
So the idea here is that this logic should exist in PP_Connext_Bridge instead? I see now that inheriting from CrosschainBase would turn this into a module which isn't correct. Slowly but surely understanding the inheritance patterns here.